chain-storage-sqlite/examples/import.rs

```diff
@@ -0,0 +1,92 @@
+extern crate cardano_storage;
```

Comment:
It seems to me this could be moved to the test directory

Reply:
No, unfortunately it can't be a test since there is no test data to apply it to.